File: python-gae/src/trigger.py
#!/usr/bin/env python
#
from google.appengine.ext import webapp
import pusher

try:
  import config
except ImportError:
  raise Exception("refer to the README on creating a config.py file")

class TriggerHandler(webapp.RequestHandler):
    def get(self):
	    p = pusher.Pusher(app_id=config.app_id, key=config.app_key, secret=config.app_secret)

	    p['my-channel'].trigger('my_event',{'msg': 'Hello world!'})

app = webapp.WSGIApplication([('/trigger', TriggerHandler)],
                                         debug=True)

File: php/excluding-recipients/src/trigger.php
<?php
require_once('../../vendor/autoload.php');
require '../../config.php';

$channel_name = 'my-channel';
$event_name = 'my_event';
$event_data = array('message' => 'Hello world!');
$socket_id = $_POST[ 'socket_id' ];

$pusher = new Pusher(APP_KEY, APP_SECRET, APP_ID);
$response = $pusher->trigger($channel_name, $event_name, $event_data, $socket_id, true);
print_r($response);
?>